Tests/test_SeqIO_features.py

```diff
@@ -362,7 +363,9 @@ def test_zero_len_dna_end(self):
     def test_simple_dna_strand0(self):
         """Feature on DNA (simple, strand 0)."""
         s = Seq("GATCRYWSMKHBVDN")
-        f = SeqFeature(FeatureLocation(5, 10), strand=0)
```

Comment:
Rather than silencing these warnings, we should just fix the calling code to move the stand argument to the location object:

```python
f = SeqFeature(FeatureLocation(5, 10, strand=0))
```

Reply:
Done.